cmd/tnf/feedback/feedback.go

```diff
@@ -0,0 +1,170 @@
+package feedback
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+)
+
+var (
+	feedbackCmd = &cobra.Command{
+		Use:   "feedback",
+		Short: "Request for customer feedback on specified topics",
+		RunE:  getFeedback,
+	}
+	colorGreen = "\033[32m"
+	colorReset = "\033[0m"
+	colorRed   = "\033[31m"
+)
+
+var topicsDB = map[string]string{
+	"observability-container-logging_1":  "[Example] Your workload is using DeploymentConfig, which does not follow the recommendation of using Deployments or StatefulSets. Why are they used?",
+	"observability-termination-policy_1": "[Example] Your application uses a port (1001) that is restricted. Could you use another one?",
+	"observability-termination-policy_2": "[Example] An SSH daemon has been found running in a Pod? That's not recommended. Why is it needed?",
+}
+
+func provideFeedback() bool {
+	prompt := promptui.Select{
+		Label:        "Would you like to provide feedback?",
+		Items:        []string{"Yes", "No"},
+		HideHelp:     true,
+		HideSelected: true,
+	}
+
+	_, result, err := prompt.Run()
+	if err != nil {
+		log.Fatalf("Prompt failed %v\n", err)
+	}
+
+	return result == "Yes"
+}
+
+func getFeedbackTopics() []string {
+	data, err := os.ReadFile("./cnf-certification-test/claim.json")
+	if err != nil {
+		log.Fatalf("Error reading claim file: %v", err)
+		return nil
+	}
+
+	r := regexp.MustCompile(`\[CQ#([^\]]+)`)
```

Comment:
Mechanism for adding a topic added to the PR description.

The `CQ#s` can be added from anywhere inside the tnf test suite providing the log gets printed to the claim.json. I guess most of them will be added if a test fails, but maybe they can also be useful in case we detected something to be flagged in the `autodiscovery` for instance.